### ISSUE TITLE
Loopback model for an asset that contains a concept is untyped

### ISSUE THREAD

**Issue by dselman** [fabric-composer/fabric-composer#311]
It seems like the concept is mapped to a LB object, so the example is just `{}` in Swagger UI.

To reproduce:

1. Run the QuickStart
2. Deploy the Bond-Network from https://github.com/fabric-composer/sample-networks/tree/master/packages/Bond-Network
3. Run the composer-rest-server as described here: https://github.com/fabric-composer/fabric-composer/tree/master/packages/composer-rest-server
4. Launch the Swagger UI and browse to the org_acme_bond_PublishBond endpoint and look at the POST verb.

```
org_acme_bond_PublishBond {
transactionId (string):

The instance identifier for this type ,
bond (object):

An instance of org.acme.bond.Bond ,
timestamp (string)
} 
```

**Comment by dselman** [comment]
This is due to the implementation of the visitConcept method here:

https://github.com/fabric-composer/fabric-composer/blob/master/packages/composer-common/lib/codegen/fromcto/loopback/loopbackvisitor.js

It should reference the type of the Concept if it has already been generated.

**Comment by dselman** [comment]
@sstone1  can you take a look please.

**Comment by sstone1** [comment]
Fixed under PR #380
